setup.py

```diff
@@ -125,20 +125,20 @@ def get_es_requirements(es_version):
                   'cli',
                   'conf'],
         install_requires=[
-          'gevent >=1.0.1, <=1.2.2', # gevent version 1.0.0 do not support pyhton 2.7.8 https://github.com/gevent/gevent/issues/513
+          'gevent >=1.0.1, <1.3', # gevent version 1.0.0 do not support pyhton 2.7.8 https://github.com/gevent/gevent/issues/513
           # ES_VERSION env var is read in order to decide which version of the python library must be installed,
           # if ES_VERSION is no provided we assume that elasticsearch 2.x will be used.
           'elasticsearch {}'.format(get_es_requirements(os.environ.get('ES_VERSION', '2'))),
-          'flask-bootstrap <=4',
+          'flask-bootstrap <4',
```

Comment:
This would force flaskBootstrap 3.x, so downgrading.
I don't think this is what we want.

Reply:
@boyska wrote:
> This would force flaskBootstrap 3.x, so downgrading.
> I don't think this is what we want.

Yeap actually the intention of https://github.com/insomnia-lab/libreant/commit/71943234c630cba1a5b41e1270dd2c330d8bcca6 was to avoid the installation of bootstrap 4 cause it will brakes a lot of stuff. The problem is that the commit was wrong :)

Reply:
I see. So that's fine.